src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/RegexCultureTests.cs

```diff
@@ -415,7 +412,6 @@ public void TestIgnoreCaseRelation()
         /// <summary>
         /// This test currently only works correctly in NonBacktracking mode.
         /// </summary>
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/60753")]
```

Comment:
do we need to run outerloop, too?

Reply:
I ran these two outerloop locally and they now passed which is why I didn't queue it initially, but shouldn't hurt to make sure it works on all platforms